### ISSUE TITLE
docs: fix MD045 - Images should have alternate text (alt text)

### ISSUE THREAD

**Issue by jbampton** [apache/apisix#3586]
# Improve Docs

This issue should be labeled as "good first issue".

## Please describe which part of docs should be improved or typo fixed

Rationale: Alternate text is important for accessibility and describes the content of an image for people who may not be able to see it.

https://www.w3.org/WAI/alt/
https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md045---images-should-have-alternate-text-alt-text

## Describe the solution you'd like

All images should have `alt` text.

To test locally first remove the `MD045` entry in the Markdownlint config file and then run `markdownlint '**/*.md'`

https://github.com/apache/apisix/blob/aea13f2a752938e8b8f97466ca7c6149954ee373/.markdownlint.yml#L34

```
doc/aws.md:28 MD045/no-alt-text Images should have alternate text (alt text)
doc/aws.md:209 MD045/no-alt-text Images should have alternate text (alt text)
doc/aws.md:210 MD045/no-alt-text Images should have alternate text (alt text)
doc/aws.md:211 MD045/no-alt-text Images should have alternate text (alt text)
doc/aws.md:217 MD045/no-alt-text Images should have alternate text (alt text)
doc/discovery.md:38 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/jwt-auth.md:91 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/jwt-auth.md:94 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/key-auth.md:61 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/key-auth.md:64 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/limit-conn.md:74 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/limit-conn.md:77 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/prometheus.md:60 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/prometheus.md:64 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/prometheus.md:89 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/prometheus.md:91 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/prometheus.md:117 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/prometheus.md:119 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/prometheus.md:121 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/response-rewrite.md:126 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/sls-logger.md:100 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/wolf-rbac.md:83 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/wolf-rbac.md:86 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/zipkin.md:73 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/zipkin.md:77 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/zipkin.md:103 MD045/no-alt-text Images should have alternate text (alt text)
doc/plugins/zipkin.md:105 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/discovery.md:38 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/jwt-auth.md:90 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/jwt-auth.md:93 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/key-auth.md:59 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/key-auth.md:62 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/limit-conn.md:67 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/limit-conn.md:70 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/prometheus.md:59 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/prometheus.md:63 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/prometheus.md:87 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/prometheus.md:89 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/prometheus.md:115 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/prometheus.md:117 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/prometheus.md:119 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/response-rewrite.md:120 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/sls-logger.md:102 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/wolf-rbac.md:83 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/wolf-rbac.md:86 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/zipkin.md:73 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/zipkin.md:77 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/zipkin.md:103 MD045/no-alt-text Images should have alternate text (alt text)
doc/zh-cn/plugins/zipkin.md:105 MD045/no-alt-text Images should have alternate text (alt text)
README_CN.md:35 MD045/no-alt-text Images should have alternate text (alt text)
README.md:37 MD045/no-alt-text Images should have alternate text (alt text)
```
